keras_nlp/models/f_net/f_net_classifier.py

```diff
@@ -55,7 +55,7 @@ class FNetClassifier(Task):
             `None`, this model will not apply preprocessing, and inputs should
             be preprocessed before calling the model.
 
-    Example usage:

```

Comment:
This looks like it is missing most of the content on the BERT classifier, may be worth another look.